x-pack/plugins/apm/server/lib/search_strategies/queries/query_histograms_generator.ts

```diff
@@ -69,8 +71,7 @@ export async function* fetchTransactionDurationHistograms(
           esClient,
           params,
           histogramRangeSteps,
-          item.fieldName,
-          item.fieldValue
+          [item]
```

Comment:
If you destructure the arguments this function become much easier to read. Right now it's pretty opaque what `[item]` is.